### PR TITLE
Ok sorry now it fixed

### PR DESCRIPTION
**Pull request by Psychotic10**

### CODE REVIEW

File: data/items.json [hunk 1]
```diff
@@ -842,7 +842,7 @@
         "name": "lucky 7",
         "aliases": ["7"],
         "emoji": "<:nypsi_7:1001553113368506538>",
-        "description": "improved chances for slots",
+        "description": "improved chances for slots 60%",
```
Comment (tekoh) [inline data/items.json:845]: why did you change the description?

Comment (Psychotic10) [reply data/items.json:845]: Idk bruh sorry idk what to do

Comment (Psychotic10) [reply data/items.json:845]: So then what do I do now Huh?
